Dockerfile

```diff
@@ -10,6 +10,7 @@ ADD src /src/src
 ADD templates /src/templates
 ADD Cargo.toml /src/
 ADD Cargo.lock /src/
+#RUN cargo generate-lockfile
```

Comment:
I don't have a local rust dev environment, so I used the container to do this. Thought I'd leave the hints behind.